coalib/settings/ConfigurationGathering.py

```diff
@@ -30,16 +31,60 @@ def aspectize_sections(sections):
     :param sections:  List of section that potentially contain aspects setting.
     :return:          The new sections.
     """
-    for section_name, section in sections.items():
-        section.aspects = extract_aspects_from_section(section)
-        if section.aspects is not None and len(section.get('bears')):
-            logging.warning("'aspects' and 'bears' configuration is detected "
-                            "in section '{}'. Aspect-based configuration will "
-                            'takes priority and will overwrite any '
-                            'explicitly listed bears'.format(section_name))
+    for _, section in sections.items():
+        if validate_aspect_config(section):
+            section.aspects = extract_aspects_from_section(section)
+        else:
+            section.aspects = None
     return sections
 
 
+def validate_aspect_config(section):
+    """
+    Validate if a section contain required setting to run in aspects mode.
+
+    :param section:         The section that potentially contain aspect
+                            setting.
+    :param acquire_setting: Method used to acquire missing setting.
```

Comment:
not needed now, if https://github.com/coala/coala/issues/4580 is being rejected.

Reply:
Forget to remove this...